### DISCUSSION TITLE
Find when a repository was deleted

### DISCUSSION THREAD

**Discussion by jfsc** [community/community#22568] [New to GitHub]
<p>Hi everyone,</p>
<p> is there some way to get the date in which a repository was deleted?</p>

**Comment by CleanCut** [comment, marked as answer]
<p><em>Update as of Sept. 30th, 2019:</em></p>
<p>A deleted repository <a href="https://help.github.com/en/articles/restoring-a-deleted-repository" rel="nofollow noopener">can be restored within 90 days</a>, unless the repository was part of a fork network that is not currently empty. A fork network consists of a parent repository, the repository’s forks, and forks of the repository’s forks. If your repository was part of a fork network, it cannot be restored unless every other repository in the network is deleted or has been detached from the network.</p>
<p>If you want to restore a repository that was part of a fork network that is not currently empty, you can <a href="https://support.github.com/contact" rel="nofollow noopener">contact GitHub Support</a>.</p>

**Reply by putra4648** [reply]
Is possible to restore repositories more than 90 days?

**Reply by CleanCut** [reply]
@putra4648 No, it is not possible.

**Comment by Harsha040526** [comment]
how can i restore that repository
